### ISSUE TITLE
getComputedStyle is globally mocked in jest builder, but it is available in jsdom

### ISSUE THREAD

**Issue by flavianh** [just-jeb/angular-builders#1333]
## Describe the Bug

The jest builder documentation [states that jsdom does not have an implementation for getComputedStyle](https://github.com/just-jeb/angular-builders/blob/master/packages/jest/README.md?plain=1#L100). However:

1. It does have one
2. The global mock was breaking my tests with angular-testing-library

## Minimal Reproduction

[Here is the implementation in jsdom](https://github.com/jsdom/jsdom/blob/22f7c3c51829a6f14387f7a99e5cdf087f72e685/lib/jsdom/browser/Window.js#L786).

## Expected Behavior

Up to the maintainers, but I see multiple possibilities:

1. The jsdom implementation is not sufficient and the globalMock should stay.
2. The jsdom implementation is sufficient and we should remove it from the globalMock. 

In both cases the README should be updated to reflect this reality.

## Environment

<pre><code>
Libs
- @angular/core version: 14.2.56
- @angular-devkit/build-angular version: 14.2.6
- @angular-builders/{the name of the builder} version: 14.1.0
- Jest version: 28.1.3
- jsdom version: 19.0.0

</code></pre>

**Comment by just-jeb** [comment]
Thanks for bringing this up! I believe it doesn't work 100%, if you look it up you can see that there are certain unsupported cases. Anyways, the history behind this is that it wasn't supported so we had a global mock and then it was implemented and we had to maintain backwards compatibility. Hence, we added `globalMocks` [option](https://github.com/just-jeb/angular-builders/tree/master/packages/jest#builder-options) that allows you to specify which global mocks you want to enable and to make it backwards compatible we enabled all of the by default.  

To sum it up: 
1. You're right, the README should be updated to reflect the reality. Would you like to make a PR?
2. In case it's breaking your tests you can disable it by simply passing `globalMocks` array without `getComputedStyle`.
3. The behavior should probably change (disable implemented mocks by default) but this would be a breaking change so we can only change it in v16. @arturovt wdyt?

**Comment by flavianh** [comment]
There you go :)

**Comment by just-jeb** [comment]
Thank you! Though it seems you created a PR to the forked master instead of this one, could you create one here please?

**Comment by flavianh** [comment]
Oops, my bad there you go https://github.com/just-jeb/angular-builders/pull/1334
